maestro-cli/src/main/java/maestro/cli/analytics/Analytics.kt

```diff
@@ -113,49 +113,53 @@ object Analytics {
+            }
 
-        if (!analyticsState.enabled) {
-            logger.trace("Analytics disabled with config file, not uploading")
-            return
-        }
+            if (!analyticsState.enabled) {
+                logger.trace("Analytics disabled with config file, not uploading")
+                return
+            }
 
-        if (!uploadConditionsMet) {
-            logger.trace("Upload conditions not met, not uploading")
-            return
-        }
+            if (!uploadConditionsMet) {
+                logger.trace("Upload conditions not met, not uploading")
+                return
+            }
 
-        val report = AnalyticsReport(
-            uuid = analyticsState.uuid,
-            freshInstall = !hasRunBefore,
-            cliVersion = EnvUtils.CLI_VERSION?.toString() ?: "Unknown",
-            os = EnvUtils.OS_NAME,
-            osArch = EnvUtils.OS_ARCH,
-            osVersion = EnvUtils.OS_VERSION,
-            javaVersion = EnvUtils.getJavaVersion().toString(),
-            xcodeVersion = IOSEnvUtils.xcodeVersion,
-            flutterVersion = EnvUtils.getFlutterVersionAndChannel().first,
-            flutterChannel = EnvUtils.getFlutterVersionAndChannel().second,
-            androidVersions = AndroidEnvUtils.androidEmulatorSdkVersions,
-            iosVersions = IOSEnvUtils.simulatorRuntimes,
-        )
+            val report = AnalyticsReport(
+                uuid = analyticsState.uuid,
+                freshInstall = !hasRunBefore,
+                cliVersion = EnvUtils.CLI_VERSION?.toString() ?: "Unknown",
+                os = EnvUtils.OS_NAME,
+                osArch = EnvUtils.OS_ARCH,
+                osVersion = EnvUtils.OS_VERSION,
+                javaVersion = EnvUtils.getJavaVersion().toString(),
+                xcodeVersion = IOSEnvUtils.xcodeVersion,
+                flutterVersion = EnvUtils.getFlutterVersionAndChannel().first,
+                flutterChannel = EnvUtils.getFlutterVersionAndChannel().second,
+                androidVersions = AndroidEnvUtils.androidEmulatorSdkVersions,
+                iosVersions = IOSEnvUtils.simulatorRuntimes,
+            )
 
-        logger.trace("Will upload analytics report")
-        logger.trace(report.toString())
+            logger.trace("Will upload analytics report")
+            logger.trace(report.toString())
 
-        try {
             ApiClient(EnvUtils.BASE_API_URL).sendAnalyticsReport(report)
             updateAnalyticsState()
         } catch (e: ConnectException) {
-            // This is fine. We don't care that much about analytics to bug user about it.
+            // This is fine. The user probably doesn't have internet connection.
+            // We don't care that much about analytics to bug user about it.
             return
+        } catch (e: Exception) {
+            // This is also fine. We don't want to bug the user.
+            // See discussion at https://github.com/mobile-dev-inc/maestro/pull/1858
```

Comment:
return?

Reply:
it's end of the function anyway, but yeah, I added the return.